Cargo.toml

```diff
@@ -20,3 +20,4 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 tar = "0.4.30"
 pico-args = "0.3.4"
+indoc = "1.0"
```

Comment:
I couldn't find any actual uses of indoc. Is it still necessary?

I would prefer not to increase the supply chain surface of the supply chain inspection tool!

Reply:
Hahaaa! Agree. Yep, I was using it and then switched to concat! but forgot to remove it here. I'll do that now.